.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    environment: NPM
```

Comment:
This environment is currently configured to only deploy from the `master` branch. I'm not sure, but this _may_ cause some issues with the version PR raised, but since it's triggered from `master` I don't think it will.